src/ServiceStack.OrmLite/Async/OrmLiteWriteCommandExtensionsAsync.cs

```diff
@@ -73,12 +73,12 @@ internal static Task<int> UpdateAsync<T>(this IDbCommand dbCmd, T obj, Cancellat
                 });
         }
 
-        internal static Task<int> UpdateAsync<T>(this IDbCommand dbCmd, CancellationToken token, params T[] objs)
+        internal static Task<int> UpdateAsync<T>(this IDbCommand dbCmd, CancellationToken token, Action<IDbCommand> commandFilter, T[] objs)
         {
```

Comment:
I removed the `params` because this is an internal method which cannot be called from anything other than some other part of OrmLite, so `params` isn't needed.